Header header

# distance sensors
bool prox_front
bool prox_right
bool prox_back
bool prox_left
